packages/devextreme/js/__internal/ui/form/form.ts

```diff
@@ -175,6 +199,7 @@ class Form extends Widget<FormProperties> {
       stylingMode: config().editorStylingMode,
       labelMode: 'outside',
       isDirty: false,
+      aiIntegration: undefined,
```

Comment:
I would prefer to remove this default. If ...super._getDefaultOptions() contains aiIntegration, we will override it with an undefined value